### ISSUE TITLE
hmm_online is not working

### ISSUE THREAD

**Issue by stanleywu111** [Cstolborg/HMMpy#1]
when I run hmm_online, the results are not correct, the estimated parameters is not equal to the true parameters

**Comment by Cstolborg** [comment]
Yes, thanks for pointing it out. I acidentally included some more modules in this version, which is not yet ready for use. Will remove them in the next update.
For now, hmmpy.jump, hmmpy.mle and hmmpy.sampler works.
